messaging/connectors/jms/src/main/java/io/helidon/messaging/connectors/jms/JmsConnector.java

```diff
@@ -116,20 +116,24 @@
     static final String TYPE_PROP_DEFAULT = "queue";
     static final String JNDI_JMS_FACTORY_DEFAULT = "ConnectionFactory";
 
+    static final String SCHEDULER_THREAD_NAME_PREFIX = "jms-poll-";
+    static final String EXECUTOR_THREAD_NAME_PREFIX = "jms-";
+
     private final Instance<ConnectionFactory> connectionFactories;
 
     private final ScheduledExecutorService scheduler;
     private final ExecutorService executor;
     private final Map<String, SessionMetadata> sessionRegister = new HashMap<>();
+    private Map<String, ConnectionFactory> connectionFactoryMap;
 
     /**
-     * Creates a new instance of JmsConnector with the required configuration.
+     * Provides a {@link io.helidon.messaging.connectors.jms.JmsConnectorBuilder} for creating
+     * a {@link io.helidon.messaging.connectors.jms.JmsConnector} instance.
      *
-     * @param config Helidon {@link io.helidon.config.Config config}
-     * @return the new instance
+     * @return new Builder instance
      */
-    public static JmsConnector create(io.helidon.config.Config config) {
-        return new JmsConnector(config, null);
+    public static JmsConnectorBuilder builder() {
```

Comment:
The `Builder` should be a static inner class of `JmsConnector`, not a standalone class - is there a reason for breaking this rule?

Reply:
no reason, fixed